addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -923,8 +945,10 @@ var SnippetsMenu = Widget.extend({
      * @param {jQuery} [$selectorChildren]
      *        elements which must have child drop zones between each of existing
      *        child
+     * @param {boolean} [forbidSanitize]
+     *        true to forbid drop into sanitized HTML model fields
      */
-    _activateInsertionZones: function ($selectorSiblings, $selectorChildren) {
+    _activateInsertionZones: function ($selectorSiblings, $selectorChildren, forbidSanitize) {
```

Comment:
Avoid a new parameter in stable, even if optional. There are other ways here, even if hacky. The easiest is probably:

Instead of
```javascript
this._activateInsertionZones($siblings, $selectorChildren, forbidSanitize);
```
Do
```javascript
this._activateInsertionZonesForbidSanitize = forbidSanitize;
this._activateInsertionZones($siblings, $selectorChildren);
delete this._activateInsertionZonesForbidSanitize;
```
That would actually make the diff smaller too I guess.

Reply:
The parameter being forwarded untouched to `_insertDropzone`, I named it `_insertDropzoneForbidSanitize` instead.